www/src/components/docs/exampleOptionForm.astro

```diff
@@ -0,0 +1,91 @@
+---
+const options = {
+  nextAuth: "NextAuth.js",
+  prisma: "Prisma",
+  tailwind: "Tailwind CSS",
+  trpc: "tRPC",
+  drizzle: "drizzle",
+};
+---
+
+<script>
+  import sdk from "@stackblitz/sdk";
+
+  function directToStackBlitz(checkedInputs: any) {
+    try {
+      sdk.openProject(
+        {
+          title: "T3 Examples",
+          description: "T3 example apps.",
+          template: "node",
+          files: {
+            "package.json": `{
+                    "name": "T3 Examples",
+                    "version": "1.0.0",
+                    "description": "My Next.js app",
+                    "scripts":{
+                        "dev": " echo 'y' | npx create-t3-app example-app --CI ${checkedInputs} && cd ./my-app  && rm ../package.json " }
+                }`,
+          },
+          settings: {
+            compile: {
+              trigger: "auto",
+              clearConsole: false,
+            },
+          },
+        },
+        {
+          newWindow: true,
+          openFile: ["package.json"],
+        },
+      );
+    } catch (error) {
+      console.error("Error", error);
+    }
+  }
+
+  function handleFormSubmit(e: Event) {
+    e.preventDefault();
+
+    const inputs = Array.from(
+      (e.target as HTMLFormElement).querySelectorAll("input"),
+    );
+    let checkedInputs = "";
+    inputs
+      .filter((input) => input.checked)
+      .map((input) => (checkedInputs += ` --${input.value}`));
+
+    directToStackBlitz(checkedInputs);
+  }
+
+  const form = document.getElementById("componentForm") as HTMLFormElement;
+
+  form.addEventListener("submit", handleFormSubmit);
+</script>
+
+<form
+  method="get"
+  id="componentForm"
+  class="mb-4 w-full rounded border border-t3-purple-300 p-4 text-lg"
+>
+  {
+    Object.entries(options).map(([code, name]) => (
+      <div>
+        <input
+          id={code}
+          type="checkbox"
+          name={name}
+          checked
```

Comment:
since drizzle + prisma isn't a valid combination, could you make it so only drizzle is checked? dont care if its a condition on the checked prop or the loop is unwrapped.

Reply:
could maybe even add validation to the form so prisma and drizzle cant be selected together, but doesn't have to be part of this pr

Reply:
Hi,
Sure, makes sense. I will keep one in this pr and then maybe make another pr where I add the other one with the conditional logic